### ISSUE TITLE
Every (2n+1)nth focus makes toolbar do not work

### ISSUE THREAD

**Issue by anfluff** [fraywing/textAngular#606]
Hi! I've found some irritating problem.
Here are steps to achieve it: 

1. focus textAngular
2. click somewhere outside of it (to unfocus)
3. focus tA again
4. now click on any button at toolbar...
5. .... tA just blurs and nothing happens. 

Im using last version 1.3.11, but actually it works on u demopage.

**Comment by andrew-makarenko** [comment]
Can't confirm (2n+1)nth as a step to reproduce, but I experience the same issue as well.

**Comment by SimeonC** [comment]
Yea, I am aware of this. It bugs me so it's fairly high priority.

**Comment by JackMorrissey-Uptake** [comment]
+1 Just ran into this as well.

**Comment by bytasv** [comment]
I'm bumping into the same issue, very annoying. Anyone has a solution?

**Comment by procterw** [comment]
I was having the same issue until I commented out the following section. Now it works on Chrome, Firefox, and Safari

element.on('blur', function(){
  if(_browserDetect.webkit) { // detect webkit
    globalContentEditableBlur = true;
  }
});

**Comment by MaximSaraev** [comment]
Hello, i have the same situation, do you fixed this or not?

**Comment by christianbartels** [comment]
Just ran into this bug too. Commenting out `globalContentEditableBlur = true;` as @procterw said fixed it for me, but I'm not aware of any side-effects this might have. This is a real show-stopper for textAngular, so please take a look at it!

**Comment by pbassut** [comment]
@procterw Please, test this on webkit based browsers and see what happens. If everything works fine, submit a PR so other people can have the same fix.

**Comment by bbourget** [comment]
From the code:
```
// fix a webkit bug, see: https://gist.github.com/shimondoodkin/1081133
// this is set true when a blur occurs as the blur of the ta-bind triggers before the click
var globalContentEditableBlur = false;
```
Which means the problem lies here somewhere:
```
		if(globalContentEditableBlur && curelement !== null){
			var isEditable = false;
			var tempEl = curelement;
			while(tempEl !== null && tempEl.tagName.toLowerCase() !== 'html' && !isEditable){
				isEditable = tempEl.contentEditable === 'true';
				tempEl = tempEl.parentNode;
			}
			if(!isEditable){
				document.getElementById('textAngular-editableFix-010203040506070809').setSelectionRange(0, 0); // set caret focus to an element that handles caret focus correctly.
				curelement.focus(); // focus the wanted element.
				if (curelement.select) {
					curelement.select(); // use select to place cursor for input elements.
				}
			}
		}
```

I am not familiar with the original issue yet, but is it possible that globalContentEditableBlur is no longer needed altogether?

**Comment by JoelParke** [comment]
I can not longer reproduce this and believe it is fixed.  Please re-open if still and issue and add a Plunkr to demonstrate.

**Comment by bbourget** [comment]
Confirmed, I can no longer reproduce as well..   Seems like it got fixed at around 1.5.2

**Comment by JoelParke** [comment]
THANKS!!!
